src/xmipp/applications/scripts/deep_micrograph_cleaner/deep_micrograph_cleaner.py

```diff
@@ -72,30 +73,22 @@ def defineParams(self):
         self.addExampleLine('xmipp_deep_micrograph_cleaner -c path/to/inputCoords/ -o path/to/outputCoords -b $BOX_SIXE  -i  /path/to/micrographs/')
         
     def run(self):
-
-      try:
-        from xmippPyModules.deepLearningToolkitUtils.utils import checkIf_tf_keras_installed, updateEnviron
-      except ImportError:
-        print("Import error")
-        print(sys.path)
-        from pylib.xmippPyModules.deepLearningToolkitUtils.utils import checkIf_tf_keras_installed, updateEnviron
-
-        checkIf_tf_keras_installed()
+        # checkIf_tf_keras_installed()
         args={}
         gpusToUse="0"
         if self.checkParam('-g'):
           gpusToUse= self.getParam('-g')
           if "None" in gpusToUse or "-1" in gpusToUse:
-            gpusToUse=None
+            gpusToUse=-1
```

Comment:
I would prefer `None`, but I guess it gets converted to `0`. Is it why you use `-1`?

Reply:
Exactly. By default it used gpu:0

Reply:
@rsanchezgarc ?

Reply:
This scripts is a wrapper to the cleanMics script, which is external to xmipp.

As long as it does not affect to the runCondaJob(cleanMics) and it is changed accordingly in the protocol, I don't mind. 

But it is important to notice that cleanMics assumes -g -1 as cpu-only

Reply:
is it ready?

Reply:
It will work as it is now, with -1 as value.